src/mmrm.cpp

```diff
@@ -54,29 +56,39 @@ Type objective_function<Type>::operator() ()
   vector<Type> y_vec_tilde = vector<Type>::Zero(y_vector.rows());
   // Sum of the log determinant will be incrementally calculated here.
   Type sum_log_det = 0.0;
+  // Get theta_size
+  int theta_size = theta.size() / n_groups;
```

Comment:
wording is confusing, because `theta_size != theta.size()`.
please change, e.g.:
```suggestion
  // Get number of variance parameters for one group.
  int theta_one_group_size = theta.size() / n_groups;
```